src/components/AutoCompleteSuggestions/BaseAutoCompleteSuggestions.tsx

```diff
@@ -64,7 +67,13 @@ function BaseAutoCompleteSuggestions<TSuggestion>(
 
     const innerHeight = CONST.AUTO_COMPLETE_SUGGESTER.SUGGESTION_ROW_HEIGHT * suggestions.length;
     const animatedStyles = useAnimatedStyle(() => StyleUtils.getAutoCompleteSuggestionContainerStyle(rowHeight.value));
-
+    const estimatedListSize = useMemo(() => {
+        const footerPadding = 40;
```

Comment:
We can save this as a constant as well just like `SUGGESTION_ROW_HEIGHT` is stored.

Reply:
Done